com.github.rigellute.spotify-tui.yaml

```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
+finish-args:
+  - "--share=network"
+  - "--filesystem=home"
+modules:
+  - name: spt
+    sources:
+    - sha256: 71e8e962bc9175003e01de1da4aafeab66d6dd7f40511ce32d734646a98b3e04
+      type: file
+      url: https://github.com/Rigellute/spotify-tui/releases/download/v0.13.0/spotify-tui-linux.tar.gz
+    - type: file
+      path: com.github.rigellute.spotify-tui.svg
+    - type: file
+      path: com.github.rigellute.spotify-tui-symbolic.svg
+    - type: file
+      path: com.github.rigellute.spotify-tui.desktop
```

Comment:
Can you get these upstreamed?

Reply:
I'll be removing these icons since I couldn't find any with correct license.

Reply:
Alright, well it needs some icon. Probably should work with upstream on that.